folio-spring-system-user/src/test/java/org/folio/spring/client/UsersClientTest.java

```diff
@@ -52,8 +53,11 @@ public void testSerialization() throws JsonProcessingException {
       new TypeReference<HashMap<String, Object>>() {}
     );
 
+    // we want someOtherProperty extracted out of extraProperties for serialization
     assertThat(user)
-      .extracting("id", "username", "active", "personal.lastName", "extraProperties.someOtherProperty")
+      .extracting("id", "username", "active", "personal.lastName", "someOtherProperty")
       .containsExactly("id", "username", true, "Smith", "value");
+
+    assertThat(user).doesNotContainKey("extraProperties");
```

Comment:
This test case was incorrect from the get-go, leading to the error not being caught.